test/methods.js

```diff
@@ -1309,3 +1342,24 @@ test( "cpfBR", function() {
 	ok( !method( "11144477715" ), "Invalid CPF Number: 1st check number failed" );
 	ok( !method( "11144477737" ), "Invalid CPF Number: 2nd check number failed" );
 } );
+
+test( "file accept - image wildcard", function() {
+	var input = acceptFileDummyInput( "test.png", "image/png" ),
+			$form = $( "<form />" ),
+			proxy = $.proxy( $.validator.methods.accept, new $.validator( {}, $form[ 0 ] ), null, input, "image/*" );
```

Comment:
That was in the initial PR (#1373) but it's defined via jQuery and essential is ES5's `Function.bind`:

https://api.jquery.com/jQuery.proxy/